### ISSUE TITLE
Add Work partOf to Advanced Search Config

### ISSUE THREAD

**Issue by clarkepeterf** [project-lux/lux-marklogic#94]
**Problem Description:**
There is no way to search for a Work being part of a Set in Advanced Search

**Expected Behavior/Solution:**
There is a new search field for Member of (or Part of) in the Advanced Search dropdown for Works

**Requirements:**
- Should this be called "Member of" ? "Part of"?
- Is the following help text good?: "Search for Works that are members of a Set."
- Questions repeated below

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

~~- [ ] Wireframe/Mockup - Mike~~
~~- [ ] Committee discussions - Sarah~~
~~- [ ] Feasibility/Team discussion - Sarah~~
~~- [ ] Backend requirements - TBD~~
~~- [ ] Frontend requirements- TBD~~
- [ ] Questions
  - should this be called "Member of" "Part of"? 
  - Is the following help text good?: "Search for Works that are members of a Set."
~~- List of questions for discussions. Answers should be documented within the issue.~~

**UAT/LUX Examples:**

~~- Example endpoints from LUX to use for development and testing, if applicable.~~

**Dependencies/Blocks:**

~~- Blocked By: Issues that are blocking the completion of the current issue.~~
~~- Blocking: Issues being blocked by the completion of the current issue.~~

**Related Github Issues:**

~~- Issues that contain similar work but are not blocking or being blocked by the current issue.~~

**Related links:**

- https://www.bugherd.com/projects/284041/tasks/2106
**Wireframe/Mockup:**
~~Place wireframe/mockup for the proposed solution at end of ticket.~~

**Comment by roamye** [comment]
@clarkepeterf, @prowns - should this be an #agenda/ticket forming item? Do we need a FE ticket for this? Who is best to answer the questions above? the team or uat?

**Comment by roamye** [comment]
TF 11/20: 

Works can be part of other works, archival sets (treated as works) are part of other archival sets. 
The new `Part Of` label set can be used to look for archival sets part of another archival sets. Since this is possible, do we want a different search term? 

Is this related to a larger conversation for Sets? Yes. 

94 is not needed for 206.

**Comment by clarkepeterf** [comment]
Repurpose into a ticket for MyCollections - each scope can be part of a set when MyCollections happens

**Comment by clarkepeterf** [comment]
@roamye  this is labeled as a Sets issue but is not needed for the Collections tab right now. However it is needed for MyCollections. Can we re-label as a MyCollections issue?

**Comment by roamye** [comment]
@clarkepeterf - I am going to remove it from this milestone and to one that is further out. Also assigned you to this.
